### DISCUSSION TITLE
1st Launch issue

### DISCUSSION THREAD

**Discussion by Manu87DS** [apache/airflow#21544] [General]
### Apache Airflow version

2.2.3 (latest released)

### What happened

Hello,

I have found the intructions to start airflow while following this nice tutorial : https://www.youtube.com/watch?v=2v9AKewyUEo&list=PLL2hlSFBmWwy6zqmbwwOhNkPpTrpbQhM5&index=1&t=2256s.

Unfortunately, after launching on my command prompt with the command "docker_compose up --build", I encountered the following bugg, that I don't understand:
![image](https://user-images.githubusercontent.com/77730417/153748003-c8630971-7a91-48cb-81bf-bcd22ee280f7.png)
Could somebody help me? Thank you by advance

### What you expected to happen

I don't know what is the problem 
Is it a problem in my docker installation or is it a problem in my OS ?

### How to reproduce

Try to follow instructions detailed in the following link : https://www.youtube.com/watch?v=2v9AKewyUEo&list=PLL2hlSFBmWwy6zqmbwwOhNkPpTrpbQhM5&index=1&t=2256s

### Operating System

Windows 10

### Deployment

Docker-Compose

### Deployment details

https://github.com/soumilshah1995/Learn-Apache-Airflow-in-easy-way-/tree/main/project

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
Converted it into discussion as this is not an issue of Airflow.

The problem is OS and specifically that you checkod out Airlfow on Windows using Windows Mounted folder. Currently Airflow only supports Running Breeze in WSL2 and only when Airflow is checked out in in "Linux" volumes - not on Windows folders that are mounted to the WSL2. Follow all the steps in https://github.com/apache/airflow/blob/main/BREEZE.rst#docker-in-wsl-2

BTW. We are working on Python version of Breeze (work is well [in progresss](https://github.com/apache/airflow/projects/13) ) and one of the goals there is to enable support for Windows + Docker combination without WSL2. But until this is complete, you shoudl be more "linux" user than "Windows" user when you are using Breeze.

**Reply by potiuk** [reply]
Even if you are not using Breeze the limitation is very similar.
